From 830085a7275c308632a18cea2f9036a5e64c52aa Mon Sep 17 00:00:00 2001
From: MaySoMusician <maysomusician@gmail.com>
Date: Thu, 6 Feb 2025 20:34:05 +0000
Subject: [PATCH] Load Google Fonts script and IBM Plex Sans JP

---
 src/app/[variants]/layout.tsx | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/src/app/[variants]/layout.tsx b/src/app/[variants]/layout.tsx
index a5fc074d..dae24cbf 100644
--- a/src/app/[variants]/layout.tsx
+++ b/src/app/[variants]/layout.tsx
@@ -29,6 +29,14 @@ const RootLayout = async ({ children, params, modal }: RootLayoutProps) => {
 
   return (
     <html dir={direction} lang={locale} suppressHydrationWarning>
+      <head>
+        <link href="https://fonts.googleapis.com" rel="preconnect" />
+        <link crossOrigin="anonymous" href="https://fonts.gstatic.com" rel="preconnect" />
+        <link
+          href="https://fonts.googleapis.com/css2?family=IBM+Plex+Sans+JP:wght@400;500;600;700&display=swap"
+          rel="stylesheet"
+        />
+      </head>
       <body>
         <GlobalProvider
           appearance={theme}
-- 
2.39.2

