From ac85e2a2986406657c0cc7fca134c57056171486 Mon Sep 17 00:00:00 2001
From: MaySoMusician <maysomusician@gmail.com>
Date: Sun, 23 Mar 2025 07:58:23 +0000
Subject: [PATCH 1/5] :bug: fix: make `node` prop in LinkProps optional

---
 src/mdx/mdxComponents/Link.tsx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mdx/mdxComponents/Link.tsx b/src/mdx/mdxComponents/Link.tsx
index 97101b1..dcda188 100644
--- a/src/mdx/mdxComponents/Link.tsx
+++ b/src/mdx/mdxComponents/Link.tsx
@@ -13,7 +13,7 @@ export interface LinkProps extends AProps {
   'data-footnote-ref'?: boolean;
   'data-link'?: string;
   'id'?: string;
-  'node': any;
+  'node'?: any;
 }
 
 const Link: FC<LinkProps> = ({ href, target, citations, ...rest }) => {
-- 
2.39.2


From 2fe0025e9e2234dc34dc7636c470a31f6b7e8363 Mon Sep 17 00:00:00 2001
From: MaySoMusician <maysomusician@gmail.com>
Date: Mon, 14 Oct 2024 10:15:38 +0000
Subject: [PATCH 2/5] Change font list for Japanese

---
 src/styles/theme/token/base.ts | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/styles/theme/token/base.ts b/src/styles/theme/token/base.ts
index 5dc5e4a..05bf5ef 100644
--- a/src/styles/theme/token/base.ts
+++ b/src/styles/theme/token/base.ts
@@ -1,9 +1,9 @@
 import { AliasToken } from 'antd/es/theme/interface';
 
 const FONT_EMOJI = `"Segoe UI Emoji","Segoe UI Symbol","Apple Color Emoji","Twemoji Mozilla","Noto Color Emoji","Android Emoji"`;
-const FONT_EN = `"HarmonyOS Sans","Segoe UI","SF Pro Display",-apple-system,BlinkMacSystemFont,Roboto,Oxygen,Ubuntu,Cantarell,"Open Sans","Helvetica Neue",sans-serif`;
-const FONT_CN = `"HarmonyOS Sans SC","PingFang SC","Hiragino Sans GB","Microsoft Yahei UI","Microsoft Yahei","Source Han Sans CN",sans-serif`;
-const FONT_CODE = `Hack,ui-monospace,SFMono-Regular,SF Mono,Menlo,Consolas,Liberation Mono,monospace`;
+const FONT_EN = `"IBM Plex Sans JP","Segoe UI","SF Pro Display",-apple-system,BlinkMacSystemFont,Roboto,Oxygen,Ubuntu,Cantarell,"Open Sans","Helvetica Neue",sans-serif`;
+const FONT_CN = ``;
+const FONT_CODE = `"Mgen+ 1mn",ui-monospace,SFMono-Regular,SF Mono,Menlo,Consolas,Liberation Mono,monospace`;
 
 export const baseToken: Partial<AliasToken> = {
   borderRadius: 5,
-- 
2.39.2


From 03d0c960e432bf6c74a97c82868e5993795a031f Mon Sep 17 00:00:00 2001
From: MaySoMusician <maysomusician@gmail.com>
Date: Sun, 20 Oct 2024 07:28:14 +0000
Subject: [PATCH 3/5] Remove FONT_CN

---
 src/styles/theme/token/base.ts | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/src/styles/theme/token/base.ts b/src/styles/theme/token/base.ts
index 05bf5ef..02e37d6 100644
--- a/src/styles/theme/token/base.ts
+++ b/src/styles/theme/token/base.ts
@@ -2,7 +2,6 @@ import { AliasToken } from 'antd/es/theme/interface';
 
 const FONT_EMOJI = `"Segoe UI Emoji","Segoe UI Symbol","Apple Color Emoji","Twemoji Mozilla","Noto Color Emoji","Android Emoji"`;
 const FONT_EN = `"IBM Plex Sans JP","Segoe UI","SF Pro Display",-apple-system,BlinkMacSystemFont,Roboto,Oxygen,Ubuntu,Cantarell,"Open Sans","Helvetica Neue",sans-serif`;
-const FONT_CN = ``;
 const FONT_CODE = `"Mgen+ 1mn",ui-monospace,SFMono-Regular,SF Mono,Menlo,Consolas,Liberation Mono,monospace`;
 
 export const baseToken: Partial<AliasToken> = {
@@ -11,6 +10,6 @@ export const baseToken: Partial<AliasToken> = {
   borderRadiusSM: 3,
   borderRadiusXS: 3,
   controlHeight: 36,
-  fontFamily: [FONT_EN, FONT_CN, FONT_EMOJI].join(','),
-  fontFamilyCode: [FONT_CODE, FONT_CN, FONT_EMOJI].join(','),
+  fontFamily: [FONT_EN, FONT_EMOJI].join(','),
+  fontFamilyCode: [FONT_CODE, FONT_EMOJI].join(','),
 };
-- 
2.39.2


From b1fcea4e1829628ab078b40ff25e1b04f57896c0 Mon Sep 17 00:00:00 2001
From: MaySoMusician <maysomusician@gmail.com>
Date: Mon, 28 Oct 2024 15:09:31 +0000
Subject: [PATCH 4/5] Do not load Chinese fonts

---
 src/ThemeProvider/index.tsx | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/src/ThemeProvider/index.tsx b/src/ThemeProvider/index.tsx
index 7d7b373..de736cd 100644
--- a/src/ThemeProvider/index.tsx
+++ b/src/ThemeProvider/index.tsx
@@ -59,15 +59,6 @@ const ThemeProvider = memo<ThemeProviderProps>(
     const webfontUrls = useMemo(
       () =>
         customFonts || [
-          genCdnUrl({ path: 'css/index.css', pkg: '@lobehub/webfont-mono' }),
-          genCdnUrl({
-            path: 'css/index.css',
-            pkg: '@lobehub/webfont-harmony-sans',
-          }),
-          genCdnUrl({
-            path: 'css/index.css',
-            pkg: '@lobehub/webfont-harmony-sans-sc',
-          }),
           genCdnUrl({ path: 'dist/katex.min.css', pkg: 'katex' }),
         ],
       [customFonts, genCdnUrl],
-- 
2.39.2


From 376b079560e232a31b2ac63e733997a09bfbd2a7 Mon Sep 17 00:00:00 2001
From: MaySoMusician <maysomusician@gmail.com>
Date: Mon, 4 Nov 2024 05:53:45 +0000
Subject: [PATCH 5/5] Temporarily ban click-to-react-component@1.1.2

cf: https://github.com/ericclemmons/click-to-component/issues/93
---
 package.json | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/package.json b/package.json
index d0d35bd..7275947 100644
--- a/package.json
+++ b/package.json
@@ -197,5 +197,8 @@
   "publishConfig": {
     "access": "public",
     "registry": "https://registry.npmjs.org"
+  },
+  "overrides": {
+    "click-to-react-component": ">=1.0.8 <1.1.2"
   }
 }
-- 
2.39.2

